<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Термопринтер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>6000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="15">
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Open file</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Burn time(us):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Image path</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>6000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Write times</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Move paper</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>16 colors</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 colors, faster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 colors, faster</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>monochrome, super fast</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
